src/Map/src/Bridge/Google/README.md

```diff
@@ -0,0 +1,75 @@
+```
+
+Available options:
+
+| Option     | Description                                                                                                                        | Default                                   |
+|------------|------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------|
+| `id`       | The id of the script tag                                                                                                           | `__googleMapsScriptId`                    |
+| `language` | Force language, see [list of supported languages](https://developers.google.com/maps/faq#languagesupport) specified in the browser | The user's preferred language             |
+| `region`   | Unicode region subtag identifiers compatible with [ISO 3166-1](https://en.wikipedia.org/wiki/ISO_3166-1)                           |                                           |
+| `nonce`    | Use a cryptographic nonce attribute                                                                                                |                                           |
+| `retries`  | The number of script load retries                                                                                                  | 3                                         |
+| `url`      | Custom url to load the Google Maps API script                                                                                      | `https://maps.googleapis.com/maps/api/js` |
+| `version`  | The release channels or version numbers                                                                                            | `weekly`                                  |
+
+## Map options
+
+With a `Map`, you can use the `GoogleOptions` class to configure the map.
+
+```php
+use Symfony\UX\Map\Bridge\Google\GoogleOptions;
+use Symfony\UX\Map\Bridge\Google\Option as GoogleOption;
+use Symfony\UX\Map\LatLng;
+use Symfony\UX\Map\Map;
+
+$map = (new Map())
+    ->center(new LatLng(48.8566, 2.3522))
+    ->zoom(6);
+
+$googleOptions = (new GoogleOptions())
+    ->mapId('YOUR_MAP_ID')
+    ->gestureHandling(GoogleOption\GestureHandling::Greedy)
+    ->backgroundColor('#f00')
+    ->doubleClickZoom(true)
+    ->zoomControl(false)
+    ->zoomControlOptions(new GoogleOption\ZoomControlOptions(
+        position: GoogleOption\ControlPosition::BLOCK_START_INLINE_END,
+    ))
+    ->mapTypeControl(false)
+    ->mapTypeControlOptions(new GoogleOption\MapTypeControlOptions(
+        mapTypeIds: [GoogleOption\MapTypeId::HYBRID, GoogleOption\MapTypeId::ROADMAP],
+        position: GoogleOption\ControlPosition::INLINE_END_BLOCK_START,
+        style: GoogleOption\MapTypeControlStyle::DROPDOWN_MENU,
+    ))
+    ->streetViewControl(false)
+    ->streetViewControlOptions(new GoogleOption\StreetViewControlOptions(
+        position: GoogleOption\ControlPosition::BLOCK_END_INLINE_START,
+    ))
```

Comment:
Not a real case, right ? Maybe focus on default case to give user reproductible steps.